### PR TITLE
Front End Features

### PR DESCRIPTION
**Pull request by tpo920**
### Changes

Describe the Pull Request here. Add any references and info to help reviewers understand your changes.

- Changed styling of drop boxes, number input boxes and layout
- Added a tool bar for future implementations
- 'Drag and drop' text dynamically changes font-family and text to selected font file

### Tests applied

What testing has been applied to validate changes are accurate and correct

- Personal testing with downloaded font files

### Issue ticket number(s)

Enter the issue numbers resolved by this pull request

-Partly addresses #27 , further layout improvements could be done

### Checklist

- [ ] Documented
- [x] Linting tests successful
- [ ] merge updated prod branch into development branch

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -36,15 +37,13 @@
...
+    margin-block-start: 1rem;
 }
 
 .vs {
     font-size: 2rem;
     font-weight: bold;
-    margin-block-end: 60px;
-    margin-left: 20px;
-    margin-right: 20px;
+    padding: 1.5rem;
```
Comment (JC4github) [inline src/App.css]: What was the reason for removing "vs" margin, the text does not seem to be centred between the two input boxes.

File: src/App.tsx [hunk 2]
```diff
@@ -34,22 +34,24 @@ function App() {
 
   return (
     <>
-      <h1>Welcome to TypeFaceOff!</h1>
+      <h1>Welcome to TypefaceOff!</h1>
```
Comment (JC4github) [inline src/App.tsx]: Why is the "f" not capitalised anymore? Will this be the same across the rest of our project?

**Review by JC4github — COMMENTED** [review]
The changes look good, I have made comments with a few questions please have a look. Great job!

File: src/App.css [hunk 3]
```diff
@@ -62,3 +61,44 @@
 .underline {
     text-decoration: underline;
 }
+
+.line-height-adjustment {
+    text-align: left;
+    margin-inline-start: 0.6em;
+    margin-block-start: 2em;
+
+    #lineHeightInputLeft {
```
Comment (jaskfla) [inline src/App.css]: Have you intentionally duplicated the styling between `#lineHeightInputLeft` and `#lineHeightInputRight`?

Perhaps they could just both have the same class 🤔

File: src/styles/FontUploader.css [hunk 4]
```diff
@@ -2,24 +2,29 @@
...
+    margin-block-start: 0.75em;
+    padding-inline-start: 6.5em;
 }
 
-.input-button {
-    margin-block-start: 10px;
-    padding-inline-start: 40px;
+.descriptor {
+    margin-block-start: 1em;
+    font-size: large;
```
Comment (jaskfla) [inline src/styles/FontUploader.css]: `large` can vary browser-to-browser. A `rem` value might help with layout predictability here.

File: src/App.tsx (hunk 2 above)
Comment (tpo920) [reply src/App.tsx]: Oh sorry, I accidently deleted the h1 tag one time and forgot the layout of the text I can change it back

**Review by jaskfla — COMMENTED** [review]
Two ID’ed elements with the same styling could perhaps just be two elements with the same class.

Otherwise looks good!

**Pull request closed without merging**